### PR TITLE
Fix getters joined terms by state

### PR DESCRIPTION
**Pull request by maikelordaz**
Related to git issue #103

### CODE REVIEW

File: contracts/facets/GettersFacet.sol [hunk 1]
```diff
@@ -62,41 +62,63 @@ contract GettersFacet is IGetters {
...
-        uint[] memory termsExpelled = new uint[](joinedTerms.length);
-        uint termscounter;
+        uint[] memory temporaryArray = new uint[](joinedTerms.length);
+        uint termsCounter;
         uint joinedTermsLength = joinedTerms.length;
+
         for (uint i; i < joinedTermsLength; ) {
             if (wasExpelled(joinedTerms[i], participant)) {
-                termsExpelled[termscounter] = i;
+                temporaryArray[termsCounter] = i;
```
Comment (MohammedTaka) [inline contracts/facets/GettersFacet.sol]: temporaryArray[termsCounter]  should be joinedTerms[i]

Comment (MohammedTaka) [reply contracts/facets/GettersFacet.sol]: @maikelordaz please confirm this is fixed

**Comment by maikelordaz** [comment]
Test flow
- Create 5 terms. 0, 1, 2, 3, 4
- Participant 1 joins terms 2, 3 and 4
- Term 3 started

For context. First table all joined terms. Second tabled initialized terms, last table active terms. All for participant 1

image

File: contracts/facets/GettersFacet.sol (hunk 1 above)
Comment (maikelordaz) [reply contracts/facets/GettersFacet.sol]: Yes. It is assigned now to the term id

**Pull request merged**